backend/app/services/alpha_vantage_client.py

```
@@ -0,0 +1,96 @@
+        return settings.alpha_vantage_api_key
+    try:
+        from app.models.database import SessionLocal, AppSetting
+        db = SessionLocal()
+        try:
+            entry = db.query(AppSetting).filter(AppSetting.key == "alpha_vantage_api_key").first()
+            if entry:
+                return entry.value
+        finally:
+            db.close()
+    except Exception:
+        pass
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)